java/src/main/java/ai/onnxruntime/OrtSession.java

```diff
@@ -495,12 +496,15 @@ public int getID() {
 
     private final List<Long> customLibraryHandles;
 
+    private Map<String, String> configEntries;
```

Comment:
>configEntries [](start = 32, length = 13)

Should this be final?

Reply:
Yes it should be (though it doesn't matter too much). I can fix that when I'm in here adding the other things from the C API, but I don't think I'll make the 1.5.0 release for that.

Reply:
Sorry I totally missed this when I merged, I can add it in my other unrelated PRs, this change is not for 1.5